db/pre-re/pet_db.conf

```diff
@@ -33,193 +33,41 @@ pet_db:(
  **************************************************************************
 {
 	// ================ Mandatory fields ==============================
-	Id: MonsterID						      (int)
-	Name: "Pet Name"                                           (string)
-	EggItem: "Egg_Item_Name"                                   (string)
+	Id: ID                               (int)
+	Name: "Pet Name"                     (string)
+	EggItem: Egg Id                      (string)
 	// ================ Optional fields ===============================
-	TamingItem: "Taming_Item_Name"                             (string)
-	FoodItem: "Food_Item_Name"                                 (string)
-	AccessoryItem: "Accessory_Item_Name"                       (string)
-	FoodEffectiveness: HungerPoints                               (int)
-	HungerDelay: IntervalDuration                                 (int)
-	HungerDecrement: HungerPoints                                 (int)
+	TamingItem: Taming Item              (string, defaults to 0)
```

Comment:
Same for some fields like TamingItem, FoodId, AccessoryItem, its better to quote them so no mistakes are made by others.

Reply:
Changed!